build.cmd

```diff
@@ -36,6 +36,7 @@ if exist "%BinlogPath%" (
         mkdir "%LogDirectory%"
     )
     move "%BinlogPath%" "%LogDirectory%"
+    echo Build Log: %LogDirectory%Build.binlog
```

Comment:
This helped in debugging an issue early on, but I feel it is nice to show where the log is if you want/need to look at it.